estimagic/differentiation/first_order_auxiliary.py

```diff
@@ -0,0 +1,68 @@
+"""
+This module contains auxiliary functions for a one step differentiation.
+"""
+
+
+def central(func, f_x0, params_sr, var, h, *func_args, **func_kwargs):
+    """
+    This function calculates the central difference.
```

Comment:
Docstring should not start with "This function calculates ..." but simply say "Calculate ..."

Reply:
Changed docstring in all three function of estimagic/differentiation/first_order_auxiliary.py .